result_test.go

```diff
@@ -140,3 +140,197 @@ func BenchmarkCreateVendorTree(b *testing.B) {
 	sm.Release()
 	os.RemoveAll(tmp) // comment this to leave temp dir behind for inspection
 }
+
+func TestStripVendor(t *testing.T) {
+	t.Run("vendor directory", stripVendorTestCase(fsTestCase{
+		before: filesystemState{
+			dirs: []fsPath{
+				fsPath{"package"},
+				fsPath{"package", "vendor"},
```

Comment:
Yep, true. I just did it this way to help visualize the package layout. I'll trim out the redundancies.